proto/storage/cve.proto

```diff
@@ -331,3 +331,9 @@ message CVSSV3 {
   float score = 12;
   Severity severity = 13;
 }
+
+message ImageCVETime {
+  string id = 1; // @gotags: search:"CVE" sql:"pk"
+  google.protobuf.Timestamp fixed_date = 4; // @gotags: search:"Fixable Since Timestamp,hidden"
+  google.protobuf.Timestamp first_image_occurrence = 9; // @gotags: search:"First Image Occurrence Timestamp,hidden"
```

Comment:
Is this the time the CVE was first discovered in this image? If so, we can make it clearer by naming it? The Occurrence timestamp is not clear enough to me.